File: fabric/src/main/java/net/cardinalboats/fabric/CardinalBoatsInit.java
package net.cardinalboats.fabric;

import net.cardinalboats.ManualSnap;
import net.cardinalboats.TurnPriming;
import net.cardinalboats.config.ModConfig;
import net.fabricmc.api.ClientModInitializer;

public class CardinalBoatsInit implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        TurnPriming.init();
        ManualSnap.init();
        ModConfig.init();
    }
}

File: platforms/fabric-1.21.11/src/main/kotlin/net/cardinalboats/CardinalBoatsInit.kt
package net.cardinalboats

import net.cardinalboats.config.CIBConfig
import net.fabricmc.api.ClientModInitializer

class CardinalBoatsInit : ClientModInitializer {
    override fun onInitializeClient() {
        TurnPriming.init()
        ManualSnap.init()
        CIBConfig.init()
    }
}
